src/ax/endpoint/validator_test.go

```diff
@@ -0,0 +1,54 @@
+package endpoint_test
+
+import (
+	"context"
+
+	. "github.com/jmalloc/ax/src/ax/endpoint"
+	"github.com/jmalloc/ax/src/internal/endpointtest"
+	"github.com/jmalloc/ax/src/internal/messagetest"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Default Validators", func() {
```

Comment:
I don't know if I've been doing this right, but all of my `Describe` calls refer to the code element they are testing, ie the struct, function or method name, so this would be organised as:

```go
Describe("BasicValidator", func() {
    Describe("Validate()", func() {
        ...
```

That's probably wrong in the world of BDD but, I'd prefer to maintain that consistency.

Reply:
Corrected.